src/index.html

```diff
@@ -15,8 +15,57 @@
       rel="stylesheet"
       href="./style.css"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Company Logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul>
+          <li>
+            <a
+              href="index.html"
+              class="link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="link">Samsung</li>
```

Comment:
```suggestion
          <li>
            <a href="#" class="link" >
              Samsung
            </a>
          </li>
```
Consider using links here

Reply:
fixed it